.github/workflows/cypress-tests.yml

```diff
@@ -56,3 +59,22 @@ jobs:
         with:
           name: screenshots-${{ needs.set-env.outputs.environment }}
           path: CypressTests/cypress/screenshots
+          
+     - name: Generate report
+        if: always()
+        run: |
```

Comment:
The `npm` tasks can't be run from the root of the project during this step because there is no package.json.

You need to tell the `run:` step what working directory to execute npm tasks in.

```yml
working-directory: CypressTests
```

see https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory

Reply:
This is still missing